lms/static/coffee/src/instructor_dashboard/util.coffee

```diff
@@ -240,10 +237,7 @@ create_email_message_views = ($messages_wrapper, emails) ->
     $email_header.append($('<input>', type: "button", name: "copy-email-body-text", value: gettext("Copy Email To Editor"), id: "copy_email_" + email_id))
 
     $close_button = $ '<a>', href: '#', class: "close-modal"
-    edx.HtmlUtils.append(
-      $close_button,
-      $('<i>', class: 'icon fa fa-times')
-    )
+    $close_button.append($('<i>', class: 'icon fa fa-times'))
```

Comment:
@ssemenova I'm not sure if you found this i tag usage in your cleanup, but it will need to be fixed.